src/recensio/plone/browser/review.py

```diff
@@ -229,15 +119,17 @@ def get_doi_url_if_active(self):
     def get_metadata(self):  # noqa: C901
         context = self.context
         meta = {}
-        fields = []  # XXX
-        for field in self.metadata_fields:
+
+        for field in self.context.metadata_fields:
             value = False  # A field is only displayed if it has a value
             is_macro = False
-            if field.startswith("get_"):
-                label = self.custom_metadata_field_labels[field]
```

Comment:
Why was custom_metadata_field_labels removed?

Reply:
because in `metadata_fields` there was only `get_journal_title` defined which was in fact `get_presentation_title`. no other `custom_metadata_field_labels` were used.